README.md

```diff
@@ -46,7 +46,8 @@ $ npm install @sap-ai-sdk/ai-api
 
 This package incorporates generative AI foundation models into your AI activities in SAP AI Core and SAP AI Launchpad.
 
-To install the  Gen AI Hub package in your project, run:
+To install the Gen AI Hub package in your project, run:
```

Comment:
[req] I think we should adhere to the official naming, which states that this should not be abbreviated and it should be lower case, so: "generative AI hub". Could you make sure this is correct **everywhere** in the repo?
[req] I would not call it "Gen AI Hub package" or similar, this is confusing. I think it would be nicer to do something like `#### Installation` instead.